reports/tasks.py

```diff
@@ -46,27 +46,19 @@
         day_of_week=TRIAL_INSTANCES_REPORT_SCHEDULE_DAY_OF_WEEK
     )
 )
-def send_trial_instances_report(recipients=settings.TRIAL_INSTANCES_REPORT_RECIPIENTS):
+def send_trial_instances_report(
+        recipients=settings.TRIAL_INSTANCES_REPORT_RECIPIENTS):
```

Comment:
@spokerman12, is there a reason you changed the formatting of this line? The old one met the code quality checks and didn't cause any issues before.

And if there is some reason to change it, you should format it according to our coding standards, like what you had to do for the django-password-validators PR.

Reply:
I ran this through autopep8, but it's true that while we should adhere to PEP8 per the handbook, the 79 character per line limit is changed to 120.